ceph.spec.in

```diff
@@ -746,7 +740,12 @@ Group:		System/Filesystems
 Requires:	ceph-base = %{_epoch_prefix}%{version}-%{release}
 Requires:	policycoreutils, libselinux-utils
 Requires(post):	ceph-base = %{_epoch_prefix}%{version}-%{release}
-Requires(post): selinux-policy-base >= %{_selinux_policy_version}, policycoreutils, gawk
```

Comment:
If we keep the original line that we will define the version to 0.0.0 then we don't need to change this at all.